docs/mergeduty/howto.md

```diff
@@ -74,6 +74,7 @@ NOTE: doing multiple gecko_migration runs is safe. Each run the script will purg
 #### connect to remote instance and get a copy of mozharness
 ```sh
 # connect to remote instance.
+# you may want to use `screen` before su...
```

Comment:
Before, or after? After would allow other to take over with all the context.

Reply:
Hm, having second thoughts here. Should we instead specify the command rather than in comments? I wonder if people would simply skip the comments and copy-paste the commands. We should add the `screen` there.

Reply:
I think it should go after, if we put in the commands to allow it. I think it won't work after a `sudo su -` without some other hackery.